build.sbt

```diff
@@ -5,6 +5,10 @@ name := "sbt-license-report"
 sbtPlugin := true
 
 publishMavenStyle := false
+bintrayOrganization := Some("sbt")
+name in bintray := "sbt-license-report"
+bintrayRepository := "sbt-plugin-releases"
+
```

Comment:
You probably want `bintrayReleaseOnPublish := false`.
The license name should also be changed to `Apache-2.0` instead of the current `Apache 2`.
Bintray also may require `scmInfo` etc, but not sure.